prwkv/rwkvrnnmodel.py

```diff
@@ -191,7 +191,8 @@ def generate(self,
                  top_p=.9,
                  top_k=20,
                  stop_on_eos=False,
-                 repetition_penalty=2.5)->Union[List[int],None]:
+                 repetition_penalty=2.5,
```

Comment:
cosmedic can you setup a linter to address issues like these?

Reply:
this was just to add the update_state parameter on the next line